indexing-service/src/test/java/org/apache/druid/indexing/seekablestream/SeekableStreamSupervisorSpecTest.java

```diff
@@ -851,6 +852,71 @@ public int getActiveTaskGroupsCount()
     autoScaler.stop();
   }
 
+  @Test
+  public void testScalingSkippedWhenPublishingTasks_EmitsCorrectSkipReason() throws InterruptedException
```

Comment:
Nit: name suggestion to separate the following: method/functionality under test, state, expected behaviour

```suggestion
  public void test_dynamicAllocationNotice_skipsScalingAndEmitsReason_ifTasksArePublishing() throws InterruptedException
```